MockHealth/src/main/resources/edm.yaml

```diff
@@ -0,0 +1,1581 @@
+propertyTypes:
```

Comment:
We agreed to no longer use `edm.yaml` for integrations. Please remove this file.